site/en/guide/autodiff.ipynb

```diff
@@ -685,7 +685,7 @@
       "source": [
         "## Control flow\n",
         "\n",
-        "Because tapes record operations as they are executed, Python control flow (using `if`s and `while`s for example) is naturally handled.\n",
+        "Because tapes record operations as they are executed, Python control flow (using `if`s and `while`s, for example) is naturally handled.\n",
```

Comment:
```suggestion
        "Because a gradient tape record operations as they are executed, Python control flow is naturally handled (for example, `if` and `while` statements).\n",
```

Reply:
@lamberta that looks good, thank you. I'd suggest to make `record` singular in this case, if you don't mind"

```suggestion
        "Because a gradient tape records operations as they are executed, Python control flow is naturally handled (for example, `if` and `while` statements).\n",
```